modelkit/assets/cli.py

```diff
@@ -119,13 +118,24 @@ def new(asset_path, asset_spec, storage_prefix, dry_run):
     response = click.prompt("[y/N]")
     if response == "y":
         with tempfile.TemporaryDirectory() as tmp_dir:
-            if asset_path.startswith("gs://"):
+            if not os.path.exists(asset_path):
+                parsed_path = parse_remote_url(asset_path)
```

Comment:
couldn't we put this logic in the `_download_object_or_prefix` method ?
it will remove some duplication between the new and update commands if we can.

Reply:
I needed to parse the prefix in order to instantiate the corresponding StorageDriver

Reply:
(Note that duplication issue will be managed with #158)